safewalk2-platform/azuredeploy.json

```diff
@@ -0,0 +1,900 @@
+            }
+        },
+		"SafewalkServerVmSize": {
+            "type": "string",
+            "defaultValue": "Standard_A1",
+            "metadata": {
+                "description": "The azure VM size that will be used for the Safewalk server. The same size will be assigned to all the Safewalk servers that will participate in the cluster."
+            }
+        },
+        "SafewalkGatewayVmSize": {
+            "type": "string",
+            "defaultValue": "Standard_A1",
+            "metadata": {
+                "description": "The azure VM size that will be used for the Safewalk Gateway."
+            }
+        },
+        "SafewalkServerRootPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The password to set for the root account of the Safewalk server. The same root password will be set to all the Safewalk servers that will participate in the cluster."
+            }
+        },
+        "SafewalkServerAdminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The password to set for the Safewalk server admin account."
+            }
+        },
+        "SafewalkGatewayRootPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The password to set for the root account of the Safewalk Gateway."
+            }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/safewalk2-platform-beta/safewalk2-platform/"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+            },
+            "defaultValue": ""
+        }
+    },
+    "variables": {
+        "vnetIDfalse": "[resourceId(parameters('vNetResourceGroup'), 'Microsoft.Network/virtualNetworks', parameters('VnetName'))]",
+        "vnetIDtrue": "[resourceId('Microsoft.Network/virtualNetworks', parameters('VnetName'))]",
+        "vnetID": "[variables(concat('vnetID', parameters('createVnet')))]",
+        "subnetRef": "[concat(variables('vnetID'), '/subnets/', parameters('subnetLANName'))]",
+        "subnetDMZRef": "[concat(variables('vnetID'), '/subnets/', parameters('subnetDMZName'))]",
+        "sizeOfDiskInGBSafewalk" :  "127",
+	    "sizeOfDiskInGBGateway" :  "20",
+        "location": "[resourceGroup().location]",

```

Comment:
this needs to be a parameter since it has to match the location of the vhds (which is also a paramter)